### DISCUSSION TITLE
请问@element-plus/icons-vue可以按需打包吗？

### DISCUSSION THREAD

**Discussion by tltby12341** [element-plus/element-plus#5664] [Q&A] [answered]
我参考了官网的实例，对element-plus用了全量和按需的方式进行打包，对比后发现两种方式下icons-vue都是全量引入的，有什么方式可以只打包需要使用的icon吗？
![1](https://user-images.githubusercontent.com/30487767/151276148-41487795-22a7-42e5-9ce2-b3588e4b744a.PNG)
![2](https://user-images.githubusercontent.com/30487767/151276153-6ac85eca-6149-44ee-96e8-294cb14e0f13.PNG)

**Comment by sxzz** [comment, marked as answer]
可以的，开启 Tree Shaking 相关配置。或者使用 unplugin-vue-components 或 [unplugin-icons](https://github.com/antfu/unplugin-icons) 等插件。

**Reply by tltby12341** [reply]
好的，感谢感谢
